src/main/resources/application.properties

```diff
@@ -24,7 +24,7 @@ alpine.data.directory=~/.dependency-track
 # Defines the path to the secret key to be used for data encryption and decryption.
 # The key will be generated upon first startup if it does not exist.
 # Default is "<alpine.data.directory>/keys/secret.key".
-# alpine.secret.key.path=/var/run/secrets/secret.key
+alpine.secret.key.path=/Users/kt/.dependency-track/keys/secret.key
```

Comment:
any specific reason for changing this path?
key is mounted on /var/run/secrets/secret.key currently. Not sure if the vault paths have been updated